src/test/java/com/google/javascript/cl2dts/dict.js

```diff
@@ -0,0 +1,13 @@
+goog.provide('dict');
+
+/**
+ * @const
+ * @dict
+ */
+dict.untyped = {};
+
+/**
+ * @dict
+ * @type {{a: Function}}
+ */
+dict.typed = {'a': function() {}};
```

Comment:
done, and added a usage for it.
note we don't have to put the bracket accessor in the .d.ts, it can just be a bare class